.classpath

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Don't believe this is the .gitignore file.

Should look like
![screen shot 2018-12-18 at 12 30 49 pm](https://user-images.githubusercontent.com/9095217/50174814-c4c91680-02c0-11e9-9945-d55e8e6c520b.png)

Delete this file and add the .gitignore from example/template project and recommit.

Reply:
![screen shot 2018-12-18 at 12 32 32 pm](https://user-images.githubusercontent.com/9095217/50174908-11145680-02c1-11e9-991c-626ff07a15e0.png)